app/cron.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+"""
+    This file is what runs every 5 minutes on the heroku scheduler
```

Comment:
Yeah, I intended to modify when the job started to put it into a processing state. Fixing that.